fe/fe-core/src/main/java/org/apache/doris/resource/Tag.java

```diff
@@ -65,6 +66,7 @@ public class Tag implements Writable {
     public static final String VALUE_DEFAULT_CLUSTER = "default_cluster";
     public static final String VALUE_DEFAULT_TAG = "default";
     public static final String VALUE_INVALID_TAG = "invalid";
+    public static final String VALUE_NODE_TYPE_TAG = "be";
```

Comment:
Actually, there are some predefined field like `TYPE_FUNCTION` and `VALUE_COMPUTATION`, they are supposed to do this feature.
So I think we can use them